src/components/PersonDetails/index.js

```diff
@@ -0,0 +1,112 @@
+import React, { useState, useEffect } from "react";
+import {
+  getPersonDetails,
+  getPersonTvCredits,
+  getPersonMovieCredits,
+} from "../shared/values";
+import { useParams } from "react-router-dom";
+import Movie from "../Movie";
+import TvShow from "../TvShow";
+import Header from "../Header";
+import Avatar from "../../img/Avatar.png";
+import CircularProgress from "@mui/material/CircularProgress";
+import Pages from "../shared/Pages";
+
+const PersonDetails = () => {
+  const { id } = useParams();
+  const [loadingMovies, setLoadingMovies] = useState(true);
+  const [loadingTvShows, setLoadingTvShows] = useState(true);
+
+  const [currentPage, setCurrentPage] = useState(1);
+  const [recordsPerPage] = useState(5);
```

Comment:
So, this value is never going to change, correct? So, it shouldn't be a state, we can simplify with:

```suggestion
  const recordsPerPage = 5;
```

Reply:
And you can put it in the values.js file, too